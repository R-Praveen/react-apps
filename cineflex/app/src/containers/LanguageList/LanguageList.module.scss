.languageList {
  margin-top: 80px;

  .heading {
    font-size: 15px;
  }

  .content {
    margin-top: 10px;
    display: flex;
  }
}
