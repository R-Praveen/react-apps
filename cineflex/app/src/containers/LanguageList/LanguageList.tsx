import React from "react";

import Container from "../../components/Container/Container";
import Language from "../../components/Language/Language";
import { CONSTANTS } from "../../constants/constants";
import styles from "./LanguageList.module.scss";

const LanguageList: React.FunctionComponent = () => {
  console.log("Component - Language List");

  const { LANGUAGES } = CONSTANTS;
  return (
    <Container
      heading="View in Other Languages"
      headingStyle={styles.heading}
      className={styles.languageList}
      contentStyle={styles.content}
    >
      {Object.values(LANGUAGES).map((literal: string, index: number) => (
        <Language key={index} literal={literal} />
      ))}
    </Container>
  );
};

export default LanguageList;
