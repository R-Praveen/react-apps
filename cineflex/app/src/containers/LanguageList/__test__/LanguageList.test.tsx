import React from "react";

import renderer from "react-test-renderer";

import LanguageList from "../LanguageList";

it("matches snapshot", () => {
  const tree = renderer.create(<LanguageList />).toJSON();
  expect(tree).toMatchSnapshot();
});
