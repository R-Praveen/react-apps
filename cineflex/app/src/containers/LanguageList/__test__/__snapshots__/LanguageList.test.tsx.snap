// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container languageList"
>
  <div
    className="heading heading"
  >
    View in Other Languages
  </div>
  <div
    className="content"
  >
    <div
      className="language"
    >
      E
    </div>
    <div
      className="language"
    >
      த
    </div>
    <div
      className="language"
    >
      త
    </div>
    <div
      className="language"
    >
      മ
    </div>
    <div
      className="language"
    >
      ह
    </div>
  </div>
</div>
`;
